webapp/channels/src/packages/mattermost-redux/src/utils/apps.ts

```diff
@@ -102,16 +102,23 @@ function cleanBindingRec(binding: AppBinding, topLocation: string, depth: number
 }
 
 export function validateBindings(bindings: AppBinding[] = []): AppBinding[] {
```

Comment:
Would you mind also changing the type signature of `bindings` to be `AppBinding[] | null` to prevent this from accidentally getting broken again in the future?